ci-operator/config/openshift/openshift-state-metrics/openshift-openshift-state-metrics-master.yaml

```diff
@@ -56,6 +56,10 @@ tests:
   commands: make doccheck
   container:
     from: src
+- as: e2e-aws
```

Comment:
I'd rather this was just `- as: e2e` to mark it [platform-agnostic][1].

[1]: https://github.com/openshift/release/tree/074cfb2eab4636c239da69063a9e90003ce25f26/ci-operator/platform-balance

Reply:
did we ever reach a resolution on that discussion? i don't recall that we did.

Reply:
Good point. Done.

Reply:
i'm still not sold on that convention and i don't think the discussion thread we had reached a conclusive decision on it.

Reply:
Sorry @bparees, I did not see your response when I made the change. LMK and I can revert back.

Reply:
please revert it for now

Reply:
Done

Reply:
> i don't think the discussion thread we had reached a conclusive decision on it.

We did not, but in the absence of a new consensus, I've been rolling forward with [the master-coded pattern][1] ;).  If you want a different master-coded pattern while we wait for consensus, just tell me what it is.

[1]: https://github.com/openshift/release/blob/074cfb2eab4636c239da69063a9e90003ce25f26/ci-operator/platform-balance/step-jobs-by-platform.py#L141

Reply:
> If you want a different master-coded pattern while we wait for consensus, just tell me what it is.

I thought I did, i said i wanted something like `e2e-agnostic` so that we wouldn't confuse "jobs that don't care" with "jobs that didn't get named properly"(among other things), but we didn't get consensus on that either so i wasn't expecting us to move ahead w/ any new pattern absent agreement of some set of stakeholders, or a decision by fiat from someone who can hand down this sort of cross-org decision such as @smarterclayton or @openshift/openshift-architects 

It feels a little like i'm bikeshedding here, or standing in the way or progress, but naming conventions for our jobs are very powerful(when we get them right), they enable us to do important things like what you're trying to do, as well was work others have been doing to shard jobs into testgrid, sippy analysis by platform, etc.  So there is a certain amount of consideration that needs to be given before defining a new pattern.

And yes, it would be better if we weren't relying on naming conventions (I think we all agree explicit metadata/tagging would be better) for this information, but we work with what we have and it's a simple system that's served us pretty well so far.

Reply:
I've just filed openshift/release#12924, pivoting to the explicit `e2e-agnostic` style naming to unblock us for getting additional platform-agnostic jobs like these annotated.